### PR TITLE
Major additions to style guide

### PR DESCRIPTION
**Pull request by EParzefall**
Creating an actual style guide. Please add things you'd like to see mentioned or protest if you don't agree with something.

### CODE REVIEW

**Comment by mmattel** [comment]
I like it - great!! 👍

If not already done, link this guide to the main readme.md page.

File: docs/style-guide.md [hunk 1]
```diff
@@ -1,20 +1,90 @@
...
+
+A panda enters a bar, eats[,] shoots and leaves.
+
+The comma decides whether the panda eats bamboo shoots or shoots  firearms.
+
+
+## Hyphenated Words
+
+In English, composite terms are rarely hyphenated except for adjectives.
+Some nouns have become one word overtime:
```
Comment (phil-davis) [inline docs/style-guide.md:55]:
```suggestion
Some nouns have become one word over time:
```
"overtime" is hours worked in excess of normal working hours :)
IMO this usage should be "over time"

File: docs/style-guide.md [hunk 2]
```diff
@@ -1,20 +1,90 @@
...
+
+The icons, the menu[,] and the tool bar are three navigation elements, but don't need a comma before "and".
+
+- As a rule of thumb: you don't need commas before "and", except in rare cases when two main clauses are connected by "and": The banana icon is yellow, and the cucumber icon is green. Other coordinating conjunctions are 'but, or, so, yet'.
+
+If you wonder whether commas are really that important, they can actually change the meaning of a sentence if only in rare cases. Look at the following example with and without the second comma:
+
+A panda enters a bar, eats[,] shoots and leaves.
+
+The comma decides whether the panda eats bamboo shoots or shoots  firearms.
```
Comment (phil-davis) [inline docs/style-guide.md:49]:
```suggestion
The comma decides whether the panda eats bamboo shoots or shoots firearms.
```

**Review by phil-davis — COMMENTED** [review]
Looks great

File: docs/style-guide.md (hunk 1 above)
Comment (EParzefall) [reply docs/style-guide.md:55]: Great catch and the perfect context for such an error.

**Review by mmattel — APPROVED** [review]

**Pull request merged**